presto-native-execution/presto_cpp/main/LinuxMemoryChecker.cpp

```diff
@@ -45,8 +65,90 @@ class LinuxMemoryChecker : public PeriodicMemoryChecker {
   }
 
   void setStatFile(std::string statFile) {
-    statFile_ = statFile;
-    LOG(INFO) << fmt::format("Changed to using memory stat file {}", statFile_);
+    memStatFile_ = statFile;
+    LOG(INFO) << fmt::format(
+        "Changed to using memory stat file {}", memStatFile_);
+  }
+
+  void setMemMaxFile(const std::string& memMaxFile) {
```

Comment:
Ideally, this is only used for testing. Let's comment that this is used for testing.